PolicyTroubleshooter/src/V1/IamCheckerGrpcClient.php

```diff
@@ -2,7 +2,7 @@
 // GENERATED CODE -- DO NOT EDIT!
 
 // Original file comments:
-// Copyright 2021 Google LLC.
```

Comment:
Do you include this file intentionally?

Reply:
No, thanks for pointing it out. Have removed the file, we'll make sure to squash and merge.